xlators/protocol/client/src/client.c

```diff
@@ -2665,6 +2665,9 @@ init(xlator_t *this)
     if (ret)
         this->fini(this);
 
+    if (conf) {
+        GF_FREE(conf);
+    }
```

Comment:
This is wrong. conf is assigned to this->private after allocating dynamic memory and setting fields of conf which will be used in future.

Reply:
Then we can free conf only in case of failure  and remove the coverity warning